### ISSUE TITLE
=~ on bash 

### ISSUE THREAD

**Issue by massimo-cassandro** [githubnext/monaspace#144]
I think that the `=~` sequence may  create confusione in bash scripts, when the `~` character is used to reference the user home dir:

```
MYPATH=~/path/to/something
```

becomes

<img width="268" alt="Screenshot 2023-11-20 alle 09 51 33" src="https://github.com/githubnext/monaspace/assets/1047169/75bf70d0-5954-4855-b0b2-31aa1103b51b">

Could it be useful to create new ligatures sets for this?

**Comment by MoshiBin** [comment]
I personally recommend against using `~` in scripts like that, as it makes it harder to quote the path, which is useful if the path possibly includes spaces or special characters.

```shell
# not ideal - breaks on spaces and some special characters
MYPATH=~/path/to/something

# not ideal - stores the actual tilde (~) character
MYPATH="~/path/to/something"

# better - more resilient against spaces and special chars
MYPATH="$HOME/path/to/something"
```

**Comment by idan** [comment]
Fixed in ca8015b27bdc6d8e1201b1ad722f4ae3c38970d0
